tex/iii.1/_0059_0060.tex

```diff
@@ -68,6 +68,8 @@ \section{Норма зиску}
 те, що цими засобами виробництва володіють не-робітники, перетворює
 робітників у найманих робітників, а не-робітників — у капіталістів.
 
+\pfbreak
```

Comment:
We can't add \pfbreak at this stage. Pfbreak is kind or chapter break, so if there is one we can replace it with \plainbreak{3}. But not add new ones.

Reply:
Thanks for the explanation. I have fixed this.